sdk/communication/Azure.Communication.PhoneNumbers/tests/PhoneNumbersClient/PhoneNumbersClientLiveTests.cs

```diff
@@ -199,7 +198,7 @@ public async Task UpdateCapabilitiesAsync()
             Assert.IsTrue(updateOperation.HasCompleted);
             Assert.IsNotNull(updateOperation.Value);
             Assert.AreEqual(number, updateOperation.Value.PhoneNumber);
-            Assert.AreEqual(200, updateOperation.GetRawResponse().Status);
+            Assert.IsTrue(IsSuccess(updateOperation.GetRawResponse().Status));
```

Comment:
Is there a way to have a log as to what the status code is in the case that this assert fails? I don't recall whether the actual value is logged by default when we do assert booleans.

Reply:
Solved as part of [3836019](https://github.com/Azure/azure-sdk-for-net/pull/34826/commits/3836019f65c293a92f044cb689e08933d1c2c6bb)